client/src/pages/EditProfile.js

```diff
@@ -0,0 +1,353 @@
+import React, { Component } from "react";
+import Grid from "@material-ui/core/Grid";
+import MenuItem from "@material-ui/core/MenuItem";
+import TextField from "@material-ui/core/TextField";
+import NavigationBar from "./Navbar";
+
+import SideNavigationBar from "./SideNavBar";
+
+import Button from "@material-ui/core/Button";
+
+import SimpleSnackbar from "./snackbar";
+import { Snackbar, IconButton } from "@material-ui/core";
+
+const initalState = {
+  firstName: "",
+  lastName: "",
+  gender: "",
+  dob: "",
```

Comment:
Can you change dob to birthDate? It would make it easier to know what the variable is